### PR TITLE
Debug new signups

### PR DESCRIPTION
**Pull request by dplanner-dev**

### CODE REVIEW

File: apps/web/models/student.py [hunk 1]
```diff
@@ -39,8 +41,10 @@ class Student(models.Model):
     unauth_session_ids = ArrayField(base_field=models.CharField(max_length=32, unique=True), default=list())
 
     def send_confirmation_link(self, request):
+        logger.debug('sending email')
         full_link = request.build_absolute_uri(
             reverse('confirmation')) + '?link=' + self.confirmation_link
+        logger.debug('build link', full_link)
         if not settings.DEBUG:
             send_mail(
```
Comment (jaismith) [inline apps/web/models/student.py:45]: does this return anything? if it does, can we log it? should return a promise and resolve with success or some failure msg, that would be helpful to see

File: apps/web/views.py [hunk 2]
```diff
@@ -37,12 +37,16 @@
...
+
 import uuid
 from google.cloud import pubsub_v1
 
+logger = logging.getLogger('layuplist.web')
+
 pub_sub_publisher = pubsub_v1.PublisherClient()
 topic_paths = {
-    'course-views': pub_sub_publisher.topic_path(os.environ['GCLOUD_PROJECT_ID'], 'course-views')
+    'course-views': pub_sub_publisher.topic_path('d-planner-data', 'course-views')
```
Comment (jaismith) [inline apps/web/views.py:49]: add note in PR desc about this? seems not debugging related

**Review by jaismith — APPROVED** [review]
small comments, overall looks good, thanks for adding

**Comment by jaismith** [comment]
whoops, don't want that issue closed by this... but noting that this is related to #50

File: apps/web/views.py (hunk 2 above)
Comment (dplanner-dev) [reply apps/web/views.py:49]: This was a "I'll fix it later thing lmao" but basically during initial migration `os.environ` couldn't read the `.env` so I just hardcoded it to get over that hump.

**Pull request merged**